packages/component-library/stories/InlineNotification.stories.tsx

```diff
@@ -17,6 +17,12 @@ const multilineText = (
     this can fit into one line of text on an average screen...
   </>
 )
+const withContentBelow = (Story: React.FunctionComponent) => (
```

Comment:
I added this so it’s possible to see how `InlineNotification` animates in and affects other content.